### PR TITLE
docs(audit-log) improve signature verification documentation

### PR DESCRIPTION
**Pull request by p0pr0ck5**

### CODE REVIEW

File: app/enterprise/1.3-x/admin-api/audit-log.md [hunk 1]
```diff
@@ -320,17 +326,32 @@ Audit log entries will now contain a field `signature`:
 
 ### Validating Signatures
 
-Record signatures can be regenerated and verified by `openssl` or other 
-cryptographic tools to confirm the validity of the signature. Re-generating the 
-signature requires serializing the record into a string format that can be 
-signed. The following is a canonical implementation written in Lua:
+Record signatures may be verified by the `openssl` utility or other cryptographic
```
Comment (h3xar0n) [inline app/enterprise/1.3-x/admin-api/audit-log.md]:
```suggestion
To verify record signatures, use the `openssl` utility or other cryptographic
```

File: app/enterprise/1.3-x/admin-api/audit-log.md [hunk 2]
```diff
@@ -320,17 +326,32 @@ Audit log entries will now contain a field `signature`:
...
+tools to confirm the validity of the signature. Signatures are generated using a
```
Comment (h3xar0n) [inline app/enterprise/1.3-x/admin-api/audit-log.md]:
```suggestion
tools that confirm the validity of the signature. The signatures are generated using a
```

File: app/enterprise/1.3-x/admin-api/audit-log.md [hunk 3]
```diff
@@ -320,17 +326,32 @@ Audit log entries will now contain a field `signature`:
...
+256-bit SHA digest. The following example demonstrates verifying the above audit
```
Comment (h3xar0n) [inline app/enterprise/1.3-x/admin-api/audit-log.md]:
```suggestion
256-bit SHA digest. The following example demonstrates how to verify the audit
```

File: app/enterprise/1.3-x/admin-api/audit-log.md [hunk 4]
```diff
@@ -320,17 +326,32 @@ Audit log entries will now contain a field `signature`:
...
+log record. First, store the record signature on disk, after stripping the Base64
```
Comment (h3xar0n) [inline app/enterprise/1.3-x/admin-api/audit-log.md]:
```suggestion
log record shown above. First, store the record signature on disk, after stripping the Base64
```

File: app/enterprise/1.3-x/admin-api/audit-log.md [hunk 5]
```diff
@@ -320,17 +326,32 @@ Audit log entries will now contain a field `signature`:
...
+encoding:
+
+```bash
+$ cat <<EOF | base64 -d > record_signature
+> l2LWYaRIHfXglFa5ehFc2j9ijfERazxisKVtJnYa+QUz2ckcytxfOLuA4VKEWHgY7cCLdn5C7uRJzE6es5V2SoOV59NOpskkr5lTt9kzao64UEw5UNOdeZYZKwyhG9Ge7IsxTK6haW0iG3a9dHqlKlwvnHZTbFM8TUV/umg8sJ1QJ/5ivXecbyHYtD5luKAI6oEgIdZPtQexRkwxlzvfR8lzeC/dDc2slSrjWRbBxNFlgfRKhDdVzVzgu8pEucgKggu67PKLkJ+bQEkxX1+Yg3czIpJyC3t6cgoggb0UNtBq1uUpswe0wdueKh6G5Gzz6XrmOjlv7zSz4gtVyEHZgg==
+> EOF
+```
+
+Next, the audit record must be transformed into its canonical format used for
+signature generation. This requires serializing the record into a string format
```
Comment (h3xar0n) [inline app/enterprise/1.3-x/admin-api/audit-log.md]:
```suggestion
signature generation. This transformation requires serializing the record into a string format
```

File: package-lock.json [hunk 6]
```diff
@@ -394,8 +393,7 @@
           "version": "4.4.0",
           "resolved": "https://registry.npmjs.org/file-type/-/file-type-4.4.0.tgz",
           "integrity": "sha1-G2AOX8ofvcboDApwxxyNul95BsU=",
-          "dev": true,
-          "optional": true
```
Comment (h3xar0n) [inline package-lock.json]: Just to check--are you running NPM install from the `package.json` on master? Our current lock file may be out of date, so these changes are a nice bonus, but I want to understand why `"optional": true` was marked in so many places.

Comment (p0pr0ck5) [reply package-lock.json]: eh i dont know why this change is present :joy:

Comment (p0pr0ck5) [reply package-lock.json]: Possibly from running `make develop`? anyhow, ill revert. my bad

**Comment by p0pr0ck5** [comment]
I've squashed the edits (thanks @GayleNeumann !) and reverted the erroneous `package.json` changes. Will merge once CI passes

**Pull request merged**